packages/nx/src/project-graph/affected/locators/npm-packages.ts

```diff
@@ -51,3 +52,17 @@ export const getTouchedNpmPackages: TouchedProjectLocator<
 
   return touched;
 };
+
+function getPossiblePackagePaths(change: JsonChange): string[] {
+  const paths = [change.path[1]];
+
+  // check if it's an alternative package e.g. npm:@myorg/package-name@1.2.3
+  // standard package versions do not contain `@`
+  const npmPackage = change.value.rhs.split('@').slice(0, -1).join('@');
```

Comment:
This seems wrong that the path does not match the name of the project. I expect it should match because the "alias"ed path is just where to get a package from.. the real name of the package is.. the property name.